blivet/devices/partition.py

```diff
@@ -377,9 +377,9 @@ def part_type_uuid(self):
         if not self.exists:
             return self.part_type_uuid_req
         else:
-            if hasattr(parted.Partition, "type_uuid") and self.parted_partition.type_uuid:
```

Comment:
I think checking whether the `parted_partition` is set with something like `hasattr(parted.Partition, "type_uuid") and and self.parted_partition and self.parted_partition.type_uuid` would be better than moving the try-except block.

Reply:
no problem, adjusted.